libavcodec/h274.h

```diff
@@ -64,4 +64,14 @@ int ff_h274_apply_film_grain(AVFrame *out, const AVFrame *in,
                              H274FilmGrainDatabase *db,
                              const AVFilmGrainParams *params);
 
+typedef struct H274SEIPictureHash {
```

Comment:
do we really need this? can we rename H265RawSEIDecodedPictureHash to H2656RawSEIDecodedPictureHash and use it?

Reply:
Compared to the HEVC and H264 codes, it is better to use h266raw. You would find the h264Raw and H265Raw has the same variable but use two structs.  The H266 uses a different variable name with the 'fg_` prefix. I checked all the other structs it seemed that FFmpge is trying to keep the raw name the same as the name in the syntax struct. That's why we prefix the struct with 'Raw'

Reply:
The other reason we cannot add the `Non-Raw` struct here is that we need to consider the encoding. The CBS can be used for both decoding and encoding. The API user of CBS can simply deliver a Raw struct so the encoding and decoding works file.

Reply:
OK, you are right.